src/view/components/common/iframe-legacy/Iframe-legacy-component.ts

```diff
@@ -91,4 +102,17 @@ export class IframeLegacyComponent extends SiteChangeListener {
             }
         }
     }
+
+    /**
+     * Force to reload the current iframe component portlet,
+     * with the specified portlet url
+     * @param url Url of the portlet to display
+     */
+    reloadIframePortlet(url: string): void {
+        if(url !== undefined && url !== '') {
+            let urlSplit = url.split('/');
+            let id = urlSplit[urlSplit.length - 1];
+            this.iframe = this.loadURL(this.routingService.getPortletURL(id) + '&in_frame=true&frame=detailFrame');
+        }
```

Comment:
that is not going to work because that url is the one we use to get real portlet url. I mean the url are like /c/site-browser and not the real one with the jsp we use in the iframe. the real url we get it from the this.routingService.getPortletURL(id)